### DISCUSSION TITLE
ease speaker GLL file

### DISCUSSION THREAD

**Discussion by Jonathan1718** [Ifsttar/I-Simpa#378] [Q&A]
Is there a way to open or convert GLL ease speaker files to use as source in the simulation ?

**Comment by nicolas-f** [comment]
Hi,

No sorry, GLL file is a closed proprietary file format, as such it can't be read by open source software.

**Comment by ArmandDuverger** [comment]
[gll-tools](https://github.com/CWBudde/gll-tools) is able to read and export GLL files. I'm trying to find how to convert their data to I-simpa directivity.
